elastic/kibana
# [SecuritySolution] Add ml.canGetMlInfo capabilities

## Summary

Fixes:
https://github.com/elastic/kibana/issues/189893
https://github.com/elastic/kibana/issues/189948

Steps to verify:

1. Open `packages/kbn-es/src/serverless_resources/project_roles/security/roles.yml`, comment feature_ml.read from t1_analyst
2. Open kibana serverless locally as t1_analyst, see links are not displayed
3. Open kibana serverless locally as t2_analyst, see the links are displayed
